packages/db/test/mysql-migration.spec.ts

```diff
@@ -0,0 +1,38 @@
+import path from "path";
+import { MySqlContainer } from "@testcontainers/mysql";
+import { drizzle } from "drizzle-orm/mysql2";
+import { migrate } from "drizzle-orm/mysql2/migrator";
+import mysql from "mysql2";
+import { describe, test } from "vitest";
+
+import * as mysqlSchema from "../schema/mysql";
+
+describe("Mysql Migration", () => {
+  test("should run successfully", async () => {
```

Comment:
Bad test name -> please choose a better name and describe what the test actually does